networking/multiple_networks/secondary_networks/configuring-ip-secondary-nwt.adoc

```diff
@@ -6,20 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The following sections give instructions and information for how to configure IP address assignments for secondary networks.
+[role="_abstract"]
+You can configure IP address assignments for secondary networks so that pods can connect to the secondary networks. 
 
 include::modules/nw-multus-ipam-object.adoc[leveloffset=+1]
 
-// Dynamic IP address assignment configuration with Whereabouts
```

Comment:
can leave this info in; someone might need it (dita will just strip, it does not break conversion); we are working on tracking these